installation/aws/VmClarity.cfn

```diff
@@ -148,102 +148,293 @@ Resources:
             03start_restart_cfn_hup:
               command: systemctl restart cfn-hup.service
         install_vmclarity:
-          packages:
-            apt:
-              docker.io: []
           files:
             "/etc/vmclarity/deploy.sh":
               content:
                 Fn::Sub: |
                   #!/bin/bash
                   set -euo pipefail
 
-                  # Create the docker network for the VMClarity services if it
-                  # doesn't exist.
-                  if docker network ls | grep vmclarity; then
-                    echo "network already exists"
-                  else
-                    docker network create vmclarity
-                  fi
-
-                  # Reload the systemd daemon to ensure that all the VMClarity
-                  # units have been detected.
-                  systemctl daemon-reload
-
-                  # Enable and start/restart exploit-db-server
-                  systemctl enable exploit-db-server.service
-                  systemctl restart exploit-db-server.service
-
-                  # Enable and start/restart trivy server
-                  systemctl enable trivy_server.service
-                  systemctl restart trivy_server.service
-
-                  # Enable and start/restart grype_server
-                  systemctl enable grype_server.service
-                  systemctl restart grype_server.service
-
-                  # Enable and start/restart freshclam mirror
-                  systemctl enable vmclarity_freshclam_mirror.service
-                  systemctl restart vmclarity_freshclam_mirror.service
+                  apt-get update
+                  apt-get install -y ca-certificates curl gnupg
+                  mkdir -p /etc/apt/keyrings
+                  chmod 755 /etc/apt/keyrings
+                  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --yes --dearmor -o /etc/apt/keyrings/docker.gpg
+                  chmod a+r /etc/apt/keyrings/docker.gpg
+                  echo \
+                    "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+                    "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+                    sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+                  apt-get update
+                  apt-get -y install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
 
                   if [ "${DatabaseToUse}" == "Postgresql" ]; then
                     # Enable and start/restart postgres
-                    systemctl enable postgres.service
-                    systemctl restart postgres.service
+                    echo "COMPOSE_PROFILES=postgres" >> /etc/vmclarity/service.env
```

Comment:
what is `COMPOSE_PROFILES`?

Reply:
Compose profiles let you enable or disable parts of the compose stack, in this case the postgres service is not installed as part of the `compose up` unless the postgres profile is enabled.
https://docs.docker.com/compose/profiles/